appvm-scripts/usrbin/qubes-run-xorg

```diff
@@ -66,23 +66,9 @@ sed -e  s/%MEM%/$MEM/ \
         > /etc/X11/xorg-qubes.conf
 
 XSESSION="/etc/X11/xinit/xinitrc"
-XORG="/usr/bin/X"
-if [ -x /usr/libexec/Xorg ]; then
-    # bypass setuid-root wrapper, we don't need root here
-    XORG=/usr/libexec/Xorg
-fi
 if [ -f /etc/X11/Xsession ]; then
     # Debian-based distro, set Xsession appropriately
     XSESSION="/etc/X11/Xsession qubes-session"
-    # Debian installs Xorg without setuid root bit, with a setuid wrapper.
-    # The wrapper is not useful for qubes, but it does not matter since
-    # we can Xorg with qubes drivers without root. But we need to call
-    # Xorg directly, not X (which is the wrapper).
-    if [ -x /usr/lib/xorg/Xorg ]; then
-        XORG="/usr/lib/xorg/Xorg"
-    else
-        XORG="/usr/bin/Xorg"
-    fi
```

Comment:
You moved setting `$XORG` to a separate script, but this script still calls `$XORG` instead of the new `qubes-xorg-wrapper`. This definitely broke sys-gui. And I'm confused why any GUI-related test passed at all...